src/ast/Relation.h

```diff
@@ -152,6 +144,16 @@ class Relation : public Node {
         return name == other.name && equal_targets(attributes, other.attributes);
     }
 
+private:
```

Comment:
Private section is defined twice. In another PR we should tidy this up.

Reply:
Thanks for the comment.  Happy to change for a specific coding standard but I usually go with the following structure, using the access tags to separate them:

```
public: //functions
protected: //functions
private: //functions
private: //data
```